internal/generator/vector/output/elasticsearch/output_conf_es_test.go

```diff
@@ -0,0 +1,96 @@
+package elasticsearch_test
+
+import (
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+	logging "github.com/openshift/cluster-logging-operator/apis/logging/v1"
+	"github.com/openshift/cluster-logging-operator/internal/generator"
+	"github.com/openshift/cluster-logging-operator/internal/generator/vector/output/elasticsearch"
+	. "github.com/openshift/cluster-logging-operator/test/matchers"
+	corev1 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var _ = Describe("Generating vector config blocks", func() {
```

Comment:
we can delete this file, and move the tests in `elasticsearch_test.go` if not duplicated.

this file is present in `fluentd` generator for historical reasons. the tests from earlier generator were moved as it is to ensure new generator passes those tests. these tests should eventually be removed from `ffluentd` generator also.

Reply:
OK.  Tests in `elasticsearch_test.go` are sufficient. We can delete `output_conf_es_test.go`